### DISCUSSION TITLE
NModbus4Core

### DISCUSSION THREAD

**Discussion by devlevinspuhl** [community/community#126812] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi 
I have created NModbus4 .net 8 version
Is that ok to use with my DSSU666 project?

**Comment by carlahall678** [comment]
It seems like you're referring to an issue or topic related to the NModbus4Core library, possibly identified by a reference number "#126812". NModbus4Core is a .NET library for implementing Modbus communication protocols.

If you have a specific question [Best Classified service provider ](https://www.classifiedlinkup.com/) or issue related to NModbus4Core #126812, please provide more details or context, and I'll do my best to assist you. Whether it's about troubleshooting, implementation, or understanding a particular feature, feel free to elaborate on your inquiry.

**Comment by carlahall678** [comment]
NModbus is a C# implementation of the Modbus protocol. Provides connectivity to Modbus slave compatible devices and applications. Supports serial ASCII, serial RTU, TCP, and UDP protocols. Although analog probes and data collection [carX Street for android](https://carxstreetapkk.com/carx-street-for-android/)  technology have a lot to offer, they also present several risks. Modbus is a digital communication protocol that overcomes many of these issues.
